### PR TITLE
test: e2e tests for CSI driver

### PR DESCRIPTION
**Pull request by helayoty**
In this PR:
- Update `aks.sh` script to create a storage account according to the [documentation](https://docs.microsoft.com/en-us/azure/aks/azure-files-volume#create-an-azure-file-share)
- Add test case for CSI driver 
- Remove `bytes` import as there is no need to read the file.

//@andyzhangx

### CODE REVIEW

**Comment by helayoty** [comment]
@andyzhangx

I followed [this](https://docs.microsoft.com/en-us/azure/aks/azure-files-volume#mount-file-share-as-an-inline-volume) sample to test the new feature, however, I got the following error which means the condition for `v.CSI` is not right.

Please advice.

**Comment by andyzhangx** [comment]
what's the error now?

File: e2e/pods_test.go [hunk 1]
```diff
@@ -54,3 +48,58 @@ func TestPodLifecycle(t *testing.T) {
...
+		cmd = kubectl("get", "--raw", "/apis/metrics.k8s.io/v1beta1/namespaces/vk-test/pods/vk-e2e-csi-driver")
+		out, err := cmd.CombinedOutput()
+		if time.Now().After(deadline) {
+			t.Fatal("failed to query pod's stats from metrics server API")
+		}
+		if err == nil {
+			t.Logf("success query metrics %s", string(out))
+			break
+		}
+		time.Sleep(10 * time.Second)
```
Comment (ryanzhang-oss) [inline e2e/pods_test.go:93]: we should avoid sleep in test in most cases.

File: e2e/pods_test.go [hunk 2]
```diff
@@ -54,3 +48,58 @@ func TestPodLifecycle(t *testing.T) {
 		t.Fatal(string(out))
 	}
 }
+
+func TestPodWithCSIDriver(t *testing.T) {
```
Comment (ryanzhang-oss) [inline e2e/pods_test.go:52]: what environment is this testing against?

**Comment by helayoty** [comment]
> what's the error now?

I discovered the error not in the feature but in the `node-cli` lib (GetSecret not working). I raised this PR to migrate from this lib to use `nodeutil` instead. #211

File: e2e/pods_test.go (hunk 2 above)
Comment (helayoty) [reply e2e/pods_test.go:52]: > what environment is this testing against?
./hack/aks.sh

File: e2e/pods_test.go (hunk 1 above)
Comment (helayoty) [reply e2e/pods_test.go:93]: removed

**Comment by helayoty** [comment]
e2e tests is running successfully now

```
vk-aci-test-aks-virtual-kubelet-aci-for-aks
time="2022-07-08T17:40:35Z" level=info msg="start creating pod vk-e2e-csi-driver" addedViaRedirty=false azure.region=westus2 azure.resourceGroup=MC_vk-aci-test-1657247926_vk-aci-test-1657247926_westus2 delayedViaRateLimit=5ms key=vk-test/vk-e2e-csi-driver method=aci.CreatePod name=vk-e2e-csi-driver namespace=vk-test node=vk-aci-test-aks operatingSystem=Linux originallyAdded="2022-07-08 17:40:35.127500086 +0000 UTC m=+3095.784236998" phase=Pending plannedForWork="2022-07-08 17:40:35.132500086 +0000 UTC m=+3095.789236998" pod=vk-e2e-csi-driver provider=azure queue=syncPodsFromKubernetes reason= requeues=0 uid=8cc13c19-c0aa-4ab6-ae6c-e1cf90bce909 watchedNamespace= workerId=23
vk-aci-test-aks-virtual-kubelet-aci-for-aks
time="2022-07-08T17:40:37Z" **level=info msg="Created pod in provider"** addedViaRedirty=false delayedViaRateLimit=5ms key=vk-test/vk-e2e-csi-driver method=createOrUpdatePod name=vk-e2e-csi-driver namespace=vk-test node=vk-aci-test-aks operatingSystem=Linux originallyAdded="2022-07-08 17:40:35.127500086 +0000 UTC m=+3095.784236998" phase=Pending plannedForWork="2022-07-08 17:40:35.132500086 +0000 UTC m=+3095.789236998" pod=vk-e2e-csi-driver provider=azure queue=syncPodsFromKubernetes reason= requeues=0 uid=8cc13c19-c0aa-4ab6-ae6c-e1cf90bce909 watchedNamespace= workerId=23
```

File: e2e/fixtures_test.go [hunk 3]
```diff
@@ -1,14 +1,10 @@
 package e2e
 
 import (
-	"embed"
 	"os"
 	"os/exec"
 )
 
-//go:embed fixtures/*
```
Comment (helayoty) [inline e2e/fixtures_test.go:9]: not used. lint pipeline is complaining about that.

**Review by mingqishao — APPROVED** [review]
look good for me

**Pull request merged**